### PR TITLE
Use javax.portlet.name in project templates

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

File: modules/sdk/project-templates-portlet/src/main/resources/archetype-resources/src/main/java/portlet/__className__Portlet.java [hunk 1]
```diff
@@ -11,6 +11,8 @@
 
 import org.osgi.service.component.annotations.Component;
 
+import ${package}.constants.${className}PortletKeys;
```
Comment (gamerson) [inline modules/sdk/project-templates-portlet/src/main/resources/archetype-resources/src/main/java/portlet/__className__Portlet.java]: Lets move this above java.*

**Comment by gamerson** [comment]
ci:stop

**Comment by lawrence-lee** [comment]
closing. will resend

**Comment by lawrence-lee** [comment]
see https://github.com/gamerson/liferay-portal/pull/126

**Pull request closed without merging**